api/src/main/java/org/openmrs/parameter/VisitSearchCriteria.java

```diff
@@ -0,0 +1,250 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.parameter;
+
+import org.openmrs.Concept;
+import org.openmrs.Location;
+import org.openmrs.Patient;
+import org.openmrs.VisitAttributeType;
+import org.openmrs.VisitType;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * The VisitSearchCriteria class encapsulates the criteria used for searching visits in the system.
+ * It includes various fields that allow filtering visits based on different parameters such as visit type,
+ * patient, location, and date ranges. It also provides options to include or exclude inactive and voided visits.
+ */
+public class VisitSearchCriteria {
```

Comment:
https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#New-Public-Methods-and-Classes

Reply:
I added `@since 2.6.8` Is it correct? or should it be `2.7.0`?

Reply:
Usually, for backported things, we list both.

Reply:
Since 2.6.8 is the latest snapshot and 2.7.0 hasn't been released, do we still need to include both? There's no gap, so I'm not sure.

Reply:
Yes.

Reply:
Sure, I'll include both. I'm still a bit unclear on the reason though, let's talk about it in our next 1:1.

Reply:
Here this is less of a big deal, but in some cases, we backport to multiple versions, so it's helpful to be able to say "this is in 2.7.0, 2.6.8, and 2.5.13". If we just said "2.5.13" that could leave the mistaken impression that its available in 2.6.0. It may be a little silly here if we only backport this to 2.6, but it also clearly documents that this was originally committed against 2.7 (which is what `master` is) and then backported.

Reply:
Makes sense. Thanks!